### ISSUE TITLE
NVDA not reading WPF Datepicker watermark text when focused on.

### ISSUE THREAD

**Issue by YuvalGrMic** [nvaccess/nvda#13210]
### Steps to reproduce:
create a basic WPF form.
use DatePicer and a basic label
`        <Label x:Name="Label"
               Grid.Row="0"
               Grid.Column="1"
               Content="Date picker label"
               Target="{Binding ElementName=DatePicker}" />
        <DatePicker x:Name="DatePicker" 
                    Grid.Row="0"
                    Grid.Column="1" />`

![image](https://user-images.githubusercontent.com/88385359/147872080-3a7e83a8-e5b2-4453-8bb7-8fb289747f1b.png)
![image](https://user-images.githubusercontent.com/88385359/147872151-d35e4669-9587-4f14-914c-d235e4642585.png)
Navigate in the form with tabs, when getting to the Datepicker textBox, the watermark "select a date" is not read and "Blank" is read instead.

### Actual behavior:
the watermark "select a date" is not read and "blank" is read instead.

### Expected behavior:
NVDA would read "select a date" (or other modified watermark messages)

#### NVDA version:
2019.3.1
#### Windows version:
Edition	Windows 10 Enterprise
Version	21H2

#### Does the issue still occur after restarting your computer?
Yes

**Comment by lukaszgo1** [comment]
Please retest with the latest version of NVDA 2021.3.1 and upload a full  example application with which the issue can be reproduced.

**Comment by YuvalGrMic** [comment]
Hey,
Reproduced with NVDA 2021.3.1 as well,
Here are the results I get from a sample app:
![image](https://user-images.githubusercontent.com/88385359/148175809-8a1f4ac1-6bea-4b0a-bfa8-e95f4448cadb.png)
![image](https://user-images.githubusercontent.com/88385359/148175813-6ae7f865-5a5f-48f6-8847-924d3d99b3a8.png)
Attached is the Zip of the App: 
[Sample_app_Date_Picker.zip](https://github.com/nvaccess/nvda/files/7812734/Sample_app_Date_Picker.zip)
at: 
Sample_app_Date_Picker.zip\Sample_app_Date_Picker\Sample_app_Date_Picker\bin\Debug\net5.0-windows\Sample_app_Date_Picker.exe
You can find the exe

**Comment by YuvalGrMic** [comment]
@feerrenrut can you please Unblock this one?

**Comment by Adriani90** [comment]
@YuvalGrMic are you still having this issue with NVDA 2024.3 Beta or 2024.2 Stable?

This issue is still blocked with needs info because we didn't have an example application to test with, so it is really difficult to investigate.
Can you provide a test application as minimal test case?

**Comment by seanbudd** [comment]
Closing as we haven't received the required information. If this is still an issue, please open a new issue with a minimal application code sample to test with
